app/api/search/specs/search.spec.js

```diff
@@ -1160,4 +1160,25 @@ describe('search', () => {
     );
     expect(resultsFound.rows.length).toBe(1);
   });
+  describe('intentful searchTerm replacement', () => {
+    it('should be able to intermix regular and straight quotes', done => {
+      userFactory.mock(undefined);
+      // https://stackoverflow.com/questions/57676477/how-to-find-the-key-of-a-value-in-a-nested-object-recursively
+      const valueInObjectDeep = (obj, value) =>
+        Object.entries(obj).some(([, v]) => {
+          if (v === value) return true;
+          if (typeof v === 'object') {
+            const f = valueInObjectDeep(v, value);
+            if (f) return true;
+          }
+        });
+      userFactory.mock(undefined);
+      spyOn(elastic, 'search').and.callFake(async () => Promise.resolve(result));
```

Comment:
I think it will be really valuable and hopefully not that complex to actually go E2E with this instead of mocking. Let me check on Monday what would be the approach and I'll get back to you.

Reply:
following up on this :)